controllers/userController.js

```diff
@@ -13,19 +15,52 @@ const signToken = (id) =>
     },
     process.env.JWT_SECRET,
     {
-      expiresIn: process.env.JWT_EXPIRES_IN,
+      expiresIn: "90d",
     }
   );
 
+const generateOTP = () => {
+  return crypto.randomInt(100000, 999999);
+};
+
+const otpExpires = new Date(Date.now() + 10 * 60 * 1000);
+
 exports.signup = catchAsync(async (req, res, next) => {
+  const otp = generateOTP();
+
+  const otpCodeMessage = `Your OTP code is ${otp}. It is valid for 10 minutes.`;
```

Comment:
You will be having plenty email to send going forward and you will be needing to send html content in most cases and not plain text like this. 

For instance, you may want to highlight some text in bold, use specific font, create table or even include image in the email content.

So it's better to use a template engine that can create nice html content for you by specifying your html content and just pass the parameters to dynamically give you content.

Check this https://handlebarsjs.com/guide/#what-is-handlebars

Reply:
will do